src/commands/svc/stop.ts

```diff
@@ -0,0 +1,49 @@
+import { CliUx, Flags } from '@oclif/core'
+import { BundleService } from '../../services/bundle-service'
+import { SvcService } from '../../services/svc-service'
+import { SvcProcessResult, ServiceTypes } from './svc-process'
+
+export default class Stop extends SvcProcessResult {
+  static strict = false
+  static description = 'Stops running enabled auxiliary services'
+
+  static examples = [
+    '<%= config.bin %> <%= command.id %> --all',
+    '<%= config.bin %> <%= command.id %> ext-service',
+    '<%= config.bin %> <%= command.id %> ext-service1 ext-service2'
+  ]
+
+  static flags = {
+    all: Flags.boolean({
+      description: 'Stops all running enabled services in the bundle descriptor'
```

Comment:
"Stops all enabled auxiliary services" is enough

Reply:
done